ql/instruments/asianoption.hpp

```diff
@@ -64,13 +64,33 @@ namespace QuantLib {
                                      std::vector<Date> fixingDates,
                                      const ext::shared_ptr<StrikedTypePayoff>& payoff,
                                      const ext::shared_ptr<Exercise>& exercise);
+
+        // Providing a constructor that takes fixings as a vector of reals, defaulting to an empty
```

Comment:
We ned to use the `/*!` ... `*/` format to let Doxygen include this in the docs.